presto-tpcds/src/main/java/com/facebook/presto/tpcds/TpcdsMetadata.java

```diff
@@ -236,7 +236,7 @@ public static double schemaNameToScaleFactor(String schemaName)
         }
 
         try {
-            return Integer.parseInt(schemaName.substring(2));
+            return Double.parseDouble(schemaName.substring(2));
```

Comment:
I made it this way because TPCH is done this way. I would keep it as it is for now and change it in separate PR in both places independently from stats.